File: cpp/minmax.cpp
/*
 * minmax.cpp
 * 
 * Copyright 2018  <>
 
 */


#include <iostream>
using namespace std;
void wypelnij(int tab[], int roz) {
    cout << "Wprowadz" << roz << "liczb:" << endl;
    for(int i=0; i<roz; i++) {
        cin >> tab[i];
    
    }
}
void drukuj(int tab[], int roz) {
    
    for(int i=0; i<roz; i++) {
        cout << tab[i] << " ";
    
    }
}

void minmax(int tab[], int roz) {
    int minimum = tab[0];
    int maksimum = tab[0];
    for(int i=0; i<roz; i++) {
        if (minimum > tab[i])
            minimum = tab[i];
        if (maksimum < tab[1])
            maksimum = tab[1];

    }
    cout << "Min.: " << minimum << endl;
    cout << "Max.: " << maksimum << endl;
}
    
    
 
    
    
    
    
    
    
    
int main(int argc, char **argv)

{
    const int rozmiar = 10;
	int tab[rozmiar];
    wypelnij(tab, rozmiar);
    drukuj(tab, rozmiar);
    cout << endl;
    minmax(tab,rozmiar);
	return 0;
}

File: python/minmax.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-
#
#  minmax.py

import random 




def losuj(ile, lmin, lmax):
    lista = []
    for i in range(ile):
        lista.append(random.randint(lmin, lmax))
    print(lista)
    return lista




def minmax():
    ile = int(input('Ile podasz liczb?:'))
    min = max = int(input('Podaj liczbe: '))
    for i in range(ile - 1):
        liczba = int(input('Podaj liczbe: ')) 
        
        if min > liczba:
            min = liczba
        if max < liczba:
            max = liczba
    return min , max
    
def minimum(liczby):
    najm = lista[0]
    for i in liczby:
        if najm > i:
            njam = i
    return najm 
    
def main(args):
    # ~najm , najw = minmax()
    # ~print('Najmnijesza: ' , najm)
    # ~print('Najwieksza: ' , najw)
    liczby = losuj(20, 0 , 50)
    print('Najmnijesza: ' , minimum(liczby))
    return 0

if __name__ == '__main__':
    import sys
    sys.exit(main(sys.argv))
